synapse/lib/cell.py

```diff
@@ -5080,6 +5061,36 @@ def getCachedSslCtx(self, opts=None, verify=None):
         key = tuple(sorted(opts.items()))
         return self._sslctx_cache.get(key)
 
+    async def _initLogBase(self):
+
+        async with s_logging.loglock:
+
+            if s_logging.logbase is not None:
+                return
+
+            s_logging.logbase = await s_base.Base.anit()
+            s_logging.logbase._fini_at_exit = True
+
+            s_logging.logbase.schedCoro(s_logging._feedLogTask())
```

Comment:
The logbase feels like something we should be able to refcount & teardown on cell fini to prevent the feedtask from running beyond the lifetime of a cell; similar to how we handle other global tasks.

Reply:
There's a bit of a "not necessarily via a single cell in one python interpreter" issue here that would need to be discussed, but def worth talking through :+1: